doc-content/enterprise-only/openshift/operator-modify-proc.adoc

```diff
@@ -10,7 +10,8 @@ To modify the environment, you must modify the YAML description of the environme
 . In the OpenShift Web console navigation panel, select *Catalog -> Installed operators* or *Operators -> Installed operators*.
 . Find the `Business Automation` operator line in the table and click `KieApp` in the line. Information about the environments that you deployed using this operator is displayed.
 . Click the name of a deployed environment. 
-. Select the *YAML* tab. A YAML source is displayed.
+. Select the *YAML* tab. A YAML source is displayed. 
```

Comment:
Probably correct as is, but is it "A YAML.." or "The YAML.."?

Reply:
I think I'd need to add a description of the exact meaning of the source with "the", and I'm not sure I have teh resources to get this formulation exactly right.